api/spec/swagger_definitions/ingestions.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+require_relative 'base_types'
+
+module Ingestions
+  class << self
+
+    # For reference
+    # def ingestion_params
+    #   params.require(:data)
+    #
+    #   attributes = [
+    #     :external_source_url,
+    #     :ingestion_type,
+    #     {
+    #       source: [
+    #         :id,
+    #         :storage, { metadata: [:filename, :size, :mime_type] }
+    #       ]
+    #     }
+    #   ]
+    #
+    #   relationships = [:text]
+    #
+    #   param_config = structure_params(
+    #     attributes: attributes,
+    #     relationships: relationships
+    #   )
+    #
+    #   params.permit(param_config)
+    # end
+
+    def request_create
```

Comment:
A sample request for ingestion via upload:
![image](https://user-images.githubusercontent.com/255909/65059700-4ec70e00-d92b-11e9-82c1-ced7f969b204.png)

A sample request for ingestion via URL:
![image](https://user-images.githubusercontent.com/255909/65059881-a1082f00-d92b-11e9-9a7b-761652747856.png)